api/src/models/team.model.js

```diff
@@ -63,6 +63,10 @@ const teamSchema = new mongoose.Schema(
       type: mongoose.SchemaTypes.ObjectId,
       ref: 'theme',
     },
+    teamImage: {
+      type: String,
+      required: false,
+    },
```

Comment:
A nice have: check that the input is base64. (Not required if it takes too much time)

Reply:
I don't know where to put this in the backend 🤣 . There is a input check for image MIME types in the file upload button itself though.

Reply:
nit: prefer renaming this to something more meaningful like `profilePic`/similar